core/migrations/20230114124352_users_night_mode_nullable.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class UsersNightModeNullable extends AbstractMigration
+{
+    public function change(): void
+    {
+        $table = $this->table('nl2_users');
+        $table->changeColumn('night_mode', 'boolean', ['length' => 1, 'null' => true, 'default' => null]);
+        $table->update();
+        $this->execute('UPDATE nl2_users SET night_mode = NULL WHERE night_mode = 0');
```

Comment:
Decided to add this in so that users are correctly reset to default - no harm in a user disabling dark mode again if they explicitly turned it off previously